lua/paq.lua

```diff
@@ -297,7 +298,7 @@ return setmetatable({
     update = function() exe_op("update", pull, vim.tbl_filter(function(pkg) return pkg.exists and not pkg.pin end, packages)) end,
     clean = function() exe_op("remove", remove, check_rm()) end,
     sync = function(self) self:clean() exe_op("sync", clone_or_pull, vim.tbl_filter(function(pkg) return pkg.status ~= "removed" end, packages)) end,
-    setup = function(self, args) for k, v in pairs(args) do cfg[k] = v end return self end,
+    setup = function(args) for k, v in pairs(args) do cfg[k] = v end return self end,
```

Comment:
`setup` is meant to be a method. That's mentioned in the docs.

`setup` returns `self` so you can place the list of packages after:

```lua
require('paq'):setup({}) {
    'foo/bar.nvim',
}
```

Reply:
[Fixed.](https://github.com/barrett-ruth/paq-nvim/commit/411b838305a21342bfa407c8718beaf29b64819c). Ready to merge.